layouts/shortcodes/attestation-letter.md

```diff
@@ -1 +1 @@
-A one-page report based on our [Executive Summary](/platform-deep-dive/pentests/reports/report-contents/#executive-summary). Suitable for external stakeholders.
\ No newline at end of file
+A one-page report suitable for external stakeholders. Includes the following:<ul><li>Executive Summary</li><li>An overall list of findings</li></ul>
```

Comment:
Instead of "An overall list of findings" we can call it "An overall findings summary table"

Reply:
Thanks @yogipetkar, I changed it to "an overall findings summary table."